web/portal/src/main/webapp/groovy/portal/webui/container/UIVRowContainer.gtmpl

```diff
@@ -30,13 +30,13 @@
 
   if (!uiPortalApp.isEditing()) {
 %>
-<div id="${uicomponent.id}" class="row ma-n3 border-box-sizing <%=uiComponentClass%>" ${cssStyle}>
+<div id="${uicomponent.id}" class="row mx-n3 border-box-sizing <%=uiComponentClass%>" ${cssStyle}>
   <%uicomponent.renderChildren();%>
 </div>
 <%
   } else {
 %>
-<div class="UIContainer row ma-n3 border-box-sizing <%=uiComponentClass%> EdittingContainer<%=uicomponent.getPermissionClasses()%>" id="${uicomponent.id}" ${cssStyle}>
+<div class="UIContainer row mx-n3 border-box-sizing <%=uiComponentClass%> EdittingContainer<%=uicomponent.getPermissionClasses()%>" id="${uicomponent.id}" ${cssStyle}>
```

Comment:
How to make the bottom margin of applications applied in addition, which should be `20px` instead of `12px` added by `col` classes in the children ? should we add a `mb-2` to make the bottom margin of the overview page same as stream by example ?

Reply:
> instead of `12px` added by `col` 

We remove that by py-0 in pages.xml when we have a 'col'


> How to make the bottom margin of applications applied in addition, which should be `20px` 
added by 
.PORTLET-FRAGMENT {
    padding-bottom: var(--allPagesApplicationSpaceBottom, 20px);
}
 since we remove the padding bottom from all the page container (.singlePageApplication)

Reply:
ok, will test it on ACC, thanks.